cdn/publicfeeds-cdn.yml

```diff
@@ -0,0 +1,194 @@
+          PolicyName: LambdaEdgePolicy
+      Tags:
+        - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
+        - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
+        - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:dev:family, Value: Dovetail }
+        - { Key: prx:dev:application, Value: Feed CDN }
+
+  OriginRequestFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        ZipFile: |
+          exports.handler = (event, context, callback) => {
+            const request = event.Records[0].cf.request;
+
+            if (request.uri.startsWith('/f/')) {
+              request.uri = request.uri.slice(2);
+            } else {
+              callback(null, { status: 404 });
+            }
+
+            callback(null, request);
+          };
+      Description: !Sub ${EnvironmentType} podcast feed CDN origin requests
+      Handler: index.handler
+      MemorySize: 128
+      Role: !GetAtt EdgeLambdaRole.Arn
+      Runtime: nodejs16.x
+      Tags:
+        - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
+        - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
+        - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:dev:family, Value: Dovetail }
+        - { Key: prx:dev:application, Value: Feed CDN }
+      Timeout: 1
+  OriginRequestFunctionVersionA:
+    Type: AWS::Lambda::Version
+    Properties:
+      FunctionName: !GetAtt OriginRequestFunction.Arn
+
+  CachePolicy:
+    Type: AWS::CloudFront::CachePolicy
+    Properties:
+      CachePolicyConfig:
+        DefaultTTL: 86400 # 1 day (default)
+        MaxTTL: 31536000 # 1 year (default)
+        MinTTL: 60
+        Name: !Sub ${AWS::StackName}-cache-policy
+        ParametersInCacheKeyAndForwardedToOrigin:
+          CookiesConfig:
+            CookieBehavior: none
+            # Cookies:
+          EnableAcceptEncodingBrotli: true
+          EnableAcceptEncodingGzip: true
+          HeadersConfig:
+            HeaderBehavior: none
+            # Headers:
+          QueryStringsConfig:
+            QueryStringBehavior: none
+            # QueryStrings:
+
+  OriginRequestPolicy:
+    Type: AWS::CloudFront::OriginRequestPolicy
+    Properties:
+      OriginRequestPolicyConfig:
+        CookiesConfig:
+          CookieBehavior: none
+          # Cookies:
+        HeadersConfig:
+          HeaderBehavior: none
+          # Headers:
+        Name: !Sub ${AWS::StackName}-origin-req-policy
+        QueryStringsConfig:
+          QueryStringBehavior: none
+          # QueryStrings:
+
+  CloudFrontDistribution:
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
+        Aliases: !If [IsProduction, publicfeeds.net, staging.publicfeeds.net]
```

Comment:
Could also make these domains configurable?  Though I doubt we'd ever set this up anywhere else.